types/journey.d.ts

```diff
@@ -0,0 +1,10 @@
+export type Journey = {
+  /** Alias used for grouping related journeys together */
+  group?: string
+  /** The text to display as an option to the user. Use a function to generate context-dependant text*/
+  text: string | object | (() => string)
+  /** A function used to check if the select journey text matches this journey (optional) */
+  matchText?: (text: string) => boolean
+  /** The main function to call to kick off the journey */
+  fn?: () => Promise<any>
```

Comment:
ok I promised no more unknowns but just make sure you do a global search for them, they're hiding in the shadows.

Reply:
Fixed here, doing a global search, looks like there are some places that need to be `any`